pkgs/development/python-modules/flask-appbuilder/default.nix

```diff
@@ -64,15 +79,10 @@ buildPythonPackage rec {
   postPatch = ''
     substituteInPlace setup.py \
       --replace "apispec[yaml]>=3.3, <4" "apispec[yaml] >=3.3" \
-      --replace "Flask>=0.12, <2" "Flask" \
-      --replace "Flask-Login>=0.3, <0.5" "Flask-Login >=0.3, <0.6" \
-      --replace "Flask-Babel>=1, <2" "Flask-Babel >=1, <3" \
-      --replace "Flask-WTF>=0.14.2, <0.15.0" "Flask-WTF" \
-      --replace "marshmallow-sqlalchemy>=0.22.0, <0.24.0" "marshmallow-sqlalchemy" \
-      --replace "Flask-JWT-Extended>=3.18, <4" "Flask-JWT-Extended>=4.1.0" \
-      --replace "PyJWT>=1.7.1, <2.0.0" "PyJWT>=2.0.1" \
-      --replace "prison>=0.2.1, <1.0.0" "prison" \
-      --replace "SQLAlchemy<1.4.0" "SQLAlchemy"
+      --replace "Flask-WTF>=0.14.2, <1.0.0" "Flask-WTF" \
+      --replace "WTForms<3.0.0" "WTForms" \
+      --replace "marshmallow-sqlalchemy>=0.22.0, <0.27.0" "marshmallow-sqlalchemy" \
+      --replace "prison>=0.2.1, <1.0.0" "prison"
```

Comment:
Nit: If we want to avoid jailbreaking too many packages, here are some relevant patches/PRs:
`apispec`: https://github.com/dpgaspar/Flask-AppBuilder/pull/1903
`Flask-WTF` & `WTForms`: should be covered by the patch above (or we can include the "correct" version ranges from that commit)
`marshmallow-sqlalchemy`: https://github.com/dpgaspar/Flask-AppBuilder/pull/1905
`prison`: I had patch in my first attempt of upgrading airflow: https://github.com/tm-drtina/nixpkgs/blob/8fcdb40f383c71894c05dfc5ab1a815d00589c97/pkgs/development/python-modules/prison/default.nix

Reply:
I would love if those got merged upstream but I can also see applying patches in the interim.